File: app/src/main/java/jgappsandgames/smartreminderslite/priority/PriorityAdapter.java
package jgappsandgames.smartreminderslite.priority;

// Java
import java.util.ArrayList;

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface;

// Save
import jgappsandgames.smartreminderssave.tasks.Task;

/**
 * PriorityAdapter
 * Created by joshua on 10/9/17.
 */
class PriorityAdapter extends TaskAdapterInterface {
    // Initializers
    PriorityAdapter(PriorityActivity activity, ArrayList<Task> tasks) {
        super(activity, activity, tasks);
    }
}

File: app/src/main/java/jgappsandgames/smartreminderslite/sort/priority/PriorityAdapter.kt
package jgappsandgames.smartreminderslite.sort.priority

// App
import jgappsandgames.smartreminderslite.adapter.TaskAdapterInterface

// Save
import jgappsandgames.smartreminderssave.tasks.Task

/**
 * PriorityAdapter
 * Created by joshua on 12/14/2017.
 *
 * Task Adapter for the Priority Lists
 */
class PriorityAdapter(activity: PriorityActivity, tasks: ArrayList<Task>): TaskAdapterInterface(activity, activity, tasks)